src/app/pages/sharing/smb/smb-form/smb-form.component.ts

```diff
@@ -70,6 +71,10 @@ export class SmbFormComponent implements OnInit {
 
   title: string = helptextSharingSmb.formTitleAdd;
 
+  createDatasetProps = {
+    acltype: DatasetAclType.Nfsv4,
```

Comment:
There is a ShareType property to be used here instead. It's like a preset.
For SMB it should be `DatasetShareType.Smb` and for NFS - `DatasetShareType.Multiprotocol`.